src/emulator/modules/SceGxm/src/SceGxm.cpp

```diff
@@ -260,6 +260,25 @@ static GLenum translate_format(SceGxmTextureFormat src) {
     }
 }
 
+static GLenum translate_primitive(SceGxmPrimitiveType primType){
+    switch (primType){
+    case SCE_GXM_PRIMITIVE_TRIANGLES:
+        return GL_TRIANGLES;
+    case SCE_GXM_PRIMITIVE_TRIANGLE_STRIP:
+        return GL_TRIANGLE_STRIP;
+    case SCE_GXM_PRIMITIVE_TRIANGLE_FAN:
+        return GL_TRIANGLE_FAN;
+    case SCE_GXM_PRIMITIVE_LINES:
+        return GL_LINES;
+    case SCE_GXM_PRIMITIVE_POINTS:
+        return GL_POINTS;
+    case SCE_GXM_PRIMITIVE_TRIANGLE_EDGES: // Todo: Implement this
+    default:
```

Comment:
Please remove the default case and make the function return the default GL_TRIANGLES. That way, Clang warns if/when not all switch cases are handled.